packages/react/src/components/Combobox/Combobox.tsx

```diff
@@ -96,6 +96,10 @@ export interface ComboboxOptions<T extends As = ComboboxElement>
    * Specifies the no results ui elements for the combobox
    */
   noResultsChildren?: React.ReactElement | React.ReactElement[];
+  /**
+   * Custom popover width with max-width as 400px
+   */
+  customPopoverWidthFlag?: boolean;
```

Comment:
This looks like a implicit behaviour for some specific case which is not maintainable in future.
Or at least it as bad not obvious property name.

Maybe it will make more sense it would be `wide?: boolean;` or `variant: DEFAULT | 'WIDE` .

Reply:
Thanks @vitaliy-p44 for suggestion.
I can update property name to wide.